packages/ui/src/atoms/Popover/Popover.tsx

```diff
@@ -0,0 +1,28 @@
+"use client"
+
+import * as PopoverPrimitive from "@radix-ui/react-popover"
+import * as React from "react"
+import { cn } from "../../utils"
+
+export const Popover = PopoverPrimitive.Root
+
+export const PopoverTrigger = PopoverPrimitive.Trigger
+
+export const PopoverContent = React.forwardRef<
+  React.ElementRef<typeof PopoverPrimitive.Content>,
+  React.ComponentPropsWithoutRef<typeof PopoverPrimitive.Content>
+>(({ className, align = "center", sideOffset = 4, ...props }, ref) => (
```

Comment:
Do not use forwardRef in React 19. Just take `ComponentPropsWithRef` and destructure ref from the props